fiftyone/core/dataset.py

```
@@ -1100,6 +1100,307 @@ def default_skeleton(self, skeleton):
+
+            # Get transform from camera_front to world via ego
+            transform = dataset.get_transform_chain(
+                "camera_front", "world", intermediate_frames=["ego"]
+            )
+        """
+        import fiftyone.core.camera as foc
+
+        if intermediate_frames is None:
+            intermediate_frames = []
+
+        frames = [source_frame] + intermediate_frames + [target_frame]
```

Ruff (0.14.10)

Consider `[source_frame, *intermediate_frames, target_frame]` instead of concatenation

Replace with `[source_frame, *intermediate_frames, target_frame]`

(RUF005)